### DISCUSSION TITLE
Languages supported in Speech to Text Streaming?

### DISCUSSION THREAD

**Discussion by saxena666** [deepgram/community#840] [General help] [answered]
Hi, can someone please advise what Languages are supported in speech to text streaming ? and how can i implement this in long conversations ?

thanks

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by saxena666** [reply]
Hi @team-deepgram  - Are there plans to add more languages to this list for support streaming transcription? (https://developers.deepgram.com/docs/models-languages-overview)

**Reply by jkroll-deepgram** [reply]
Hi @sabysaxena, which languages are you looking for that we don't yet support?

**Reply by saxena666** [reply]
Persian/Farsi, Arabic, Cantonese and Tamil.

**Reply by jkroll-deepgram** [reply]
@sabysaxena We do not have Farsi or Arabic on our roadmap. We are aiming to launch Nova-2 Cantonese within the next few months. We don't support Tamil on our Nova-2 model, but we do on our Enhanced model.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @sabysaxena, all of our [languages](https://developers.deepgram.com/docs/models-languages-overview) support streaming transcription. Check out our [getting started guide](https://developers.deepgram.com/docs/getting-started-with-live-streaming-audio) for streaming, which shows usage patterns and code samples from our SDKs.
